File: exp/exponential.java
class exponential {
    static public int power(int x, int y){
        int omega=x;
        for (int i = 0; i < y-1; i++) {
            omega *=x;
        }
        return omega;
    }
    public static void main(String[] args) {
        System.out.println(power(5,3));        
    }    
}

File: exp/exponential.cpp
#include <iostream>
using namespace std;
int power(int x, int y){
    int omega =x;
    for (int i = 0; i < y-1; i++){
        omega*=x;
    }
    return omega;
}
int main()
{
    cout<<power(5,3);
    return 0;
}
